en/reports/vivliostyle-report-2020/vf2020report.md

```diff
@@ -0,0 +1,263 @@
+- 2020/6/11: [2019 Activity Report released.](https://vivliostyle.github.io/vivliostyle_doc/en/reports/vivliostyle-report-2019/vf2019report.html)
+- 2020/8/19: [Creator Book Tutorial Guide Released.](https://vivliostyle.org/make-books-with-create-book/)
+- 2020/8/29: [Announced the start of accepting donations in Japanese yen by credit card.](https://vivliostyle.org/blog/2020/11/20/You-can-support-Vivliostyle-without-a-GitHub-account/)
+
+In particular, the appeal for donations is important for a general incorporated association that does not make a profit (and does not distribute profits). In fact, we were able to collect 61,209 yen in total this fiscal year (Fig. 1), which is about 4% of our total ordinary income. This is still not a large amount, but we would like to continue to nurture it as a receptacle for people who support our activities.
+
+![Figure 1: Breakdown of ordinary income for the current fiscal year](./img/fig-1.png){ width=50% }
+ 
+### 3. Product Development Status
+
+In this section, we will explain the development status of each product for this fiscal year.
+
+#### [Vivliostyle Core / Viewer (vivliostyle.js)](https://github.com/vivliostyle/vivliostyle.js)
+
+We were able to add many functions to Vivliostyle Core and Vivliostyle Viewer (Vivliostyle.js), which are the foundation of the entire product, this year. As a result, we were able to promote support for [CSS Paged Media](https://www.w3.org/TR/css-page-3/), which had not been sufficiently supported in the past. The main features we added are as follows.
+
+ - Implementation of a slide bar for page navigation, and support for mouse wheel. ([v2.2.0 / 2020/11/26](https://github.com/vivliostyle/vivliostyle.js/releases/tag/v2.2.0))
+- Support for specifying EPUB spread view. ([v2.3.0 / 2020/12/7](https://github.com/vivliostyle/vivliostyle.js/releases/tag/v2.3.0))
+- Support for named strings to realize headers, footers, etc. ([v2.4.0 / 2020/12/28](https://github.com/vivliostyle/vivliostyle.js/releases/tag/v2.4.0))
+- Support for n-th page selector. ([v2.5.0 / 2021/2/26](https://github.com/vivliostyle/vivliostyle.js/releases/tag/v2.5.0))
+- 印刷ボタンの追加 ([v2.6.0 / 2021/3/14](https://github.com/vivliostyle/vivliostyle.js/releases/tag/v2.6.0))
+
+#### [Vivliostyle CLI](https://github.com/vivliostyle/vivliostyle-cli)
+
+This is a converter that allows CSS typesetting with a command line interface. With the addition of support for Markdown format input this fiscal year, the scope of its use has expanded dramatically. In addition, since the preview screen has been integrated into Vivliostyle Viewer, it can be said that Vivliostyle Viewer has grown into a product that allows users to read long texts, not just to check the formatting results. As will be discussed later, this is expected to have a positive impact on the future Vivliostyle Pub v1. The details for each release are as follows.
```

Comment:
> In addition, since the preview screen has been integrated into Vivliostyle Viewer, it can be said that Vivliostyle Viewer has grown into a product that allows users to read long texts, not just to check the formatting results.

この英文では、「Vivliostyle Viewerは、単に組版結果を確認するだけでなく、長文を読むことができる製品に成長した」という意味になるが、主語が間違っているのでないか？　ここでの「製品」はVivliostyle ViewerではなくてVivliostyle CLIのはずです。

元の日本語は、
> また、プレビュー画面がVivliostyle Viewerに統合されたことで、単に組版結果を確認できるだけでなく「長文を読める」プロダクトに成長したと言える。

でした。

英文の修正案：
> In addition, with the integration of Vivliostyle Viewer for the preview mode, it has grown into a product that that allows users to read long texts, not just to check the formatting results.